src/www/messages/en.json

```diff
@@ -41,6 +41,7 @@
   "licenses-page-title": "Licenses",
   "non-system-vpn-warning-detail": "Most browsers connect automatically with Outline, some do not.",
   "non-system-vpn-warning-title": "Verify your browser connection",
+  "tray-open-window": "Open",
```

Comment:
please add the string to `resources/original_messages.json`.

Reply:
Done